pages/linux/hostid.md

```diff
@@ -0,0 +1,7 @@
+# hostid
+
+> Prints the numeric identifier for the current host (not necessarily the IP address).
```

Comment:
I think "(not necessarily the IP address)" is not required. Let us keep the heading short and simple